examples/core/Graphormer/train.py

```diff
@@ -392,13 +393,16 @@ def collate(self, samples):
         node_feat = []
         in_degree, out_degree = [], []
         path_data = []
+        # use -1 padding since shortest_dist returns -1 for unreachable node pairs
         dist = -th.ones(
             (num_graphs, max_num_nodes, max_num_nodes), dtype=th.long
         )
 
         for i in range(num_graphs):
+            # positions outside num_nodes should be masked with non-zero values
```

Comment:
Is `attn_mask` a binary mask? Then perhaps just say this is a binary mask and there is no need to say "with non-zero values".

Reply:
Yes, I will refine it.